src/engine/strat_engine/metadata.rs

```diff
@@ -504,29 +504,29 @@ mod mda {
         }
 
         /// The last update time for these MDA regions
-        pub fn last_update_time(&self) -> Option<&Timespec> {
+        pub fn last_update_time(&self) -> Option<&DateTime<Local>> {
             self.mdas[self.newer()].as_ref().map(|h| &h.last_updated)
         }
     }
 
     #[derive(Debug)]
     pub struct MDAHeader {
-        last_updated: Timespec,
+        last_updated: DateTime<Local>,
 
         /// Size of region used for pool metadata.
         used: Bytes,
 
         data_crc: u32,
     }
 
-    // Implementing Default explicitly because Timespec does not implement
+    // Implementing Default explicitly because DateTime<Local> does not implement
```

Comment:
You can drop the second sentence in this comment, because this PR will have made it obsolete! Do you think there might be any point in submitting a PR against the chrono crate to implement Default for ```DateTime<Local>```.